ocaml/xapi/dbsync_slave.ml

```diff
@@ -49,7 +49,7 @@ let create_localhost ~__context info =
 	~hostname:info.hostname ~address:ip 
 	~external_auth_type:"" ~external_auth_service_name:"" ~external_auth_configuration:[] 
 	~license_params:[] ~edition:"free" ~license_server:["address", "localhost"; "port", "27000"]
-	~local_cache_sr:Ref.null ~chipset_info:[]
+	~local_cache_sr:Ref.null ~chipset_info:[] ~virt_hw_vns:[]
```

Comment:
Since here a host is creating a DB entry for itself, it would be reasonable for it to specify `virt_hw_vns:[1]` or something like `virt_hw_vns:(xapi_globs.host_virt_hw_vns)` but I have not finished reading the whole pull-request so I might update or delete this comment.